tests/test_samples.py

```diff
@@ -885,3 +885,17 @@ def test_plotting_with_integer_names():
     assert_array_equal(samples_1.loc[:, 0], samples_1.iloc[:, 0])
     with pytest.raises(KeyError):
         samples_1['0']
+
+
+def test_credibility_interval():
+    np.random.seed(3)
+    samples = NestedSamples(root='./tests/example_data/pc')
+    assert np.allclose(samples.credibility_interval("x0", level=0.68,
+                                                    method="iso-probability"),
+                       [-0.1001, 0.0951], atol=0.001)
```

Comment:
Is it possible to change these to the 'correct' values of -0.1 and 1.1, with an appropriate atol (rather than the noisy versions here)?